webpack.config.production.js

```diff
@@ -0,0 +1,41 @@
+const webpack = require("webpack");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
+const ZipPlugin = require("zip-webpack-plugin");
+
+const devWebpackConfig = require("./webpack.config");
+
+const isPrerendering = process.env.SCRIVITO_PRERENDER;
+
+function webpackConfig(env = {}) {
+  const devConfig = devWebpackConfig({ ...env, production: true });
+
+  return {
+    ...devConfig,
```

Comment:
I think we should also _not_ make use of `devServer` in production/prerendering.

Reply:
Yeah. We'll lose a simple way to test changes on IE11, but I guess nobody will miss that.